src/dotnet/ProcessStartInfoExtensions.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using System.Diagnostics;
+
+namespace Microsoft.DotNet.Cli
+{
+    internal static class ProcessStartInfoExtensions
+    {
+        public static int Execute(this ProcessStartInfo startInfo)
+        {
+            if (startInfo == null)
```

Comment:
since this is an extension method we can't have any assumptions which means we should do the null check